ShortURLBuilderProvider/Sources/ShortURLBuilderProvider/Config.swift

```diff
@@ -0,0 +1,19 @@
+enum Config {
+    static let appGroup = "" 
```

Comment:
@tanterry 

Seems that we need to have permission to access keychain if we use it in the action 🤔 
I put a placeholder here.
Not sure what should I put to test